plugins/bundle/plugin_test.go

```diff
@@ -41,6 +43,7 @@ func TestPluginOneShot(t *testing.T) {
 	bundleName := "test-bundle"
 	plugin.status[bundleName] = &Status{Name: bundleName, Metrics: metrics.New()}
 	plugin.downloaders[bundleName] = download.New(download.Config{}, plugin.manager.Client(""), bundleName)
+	plugin.loggers[bundleName] = sdk.NewStandardLogger()
```

Comment:
Nit: Having to instantiate loggers everywhere the plugin is intantiated is a suboptimal. If we exposed logging functions that accepted logger parameters, we could check for nil in the logging function and avoid this kind of thing.

Reply:
Yeah this would be avoided if these tests instantiated via the New constructor.. but 116da7809afbed2344a7c13bb8aa440eea928203 removes the need for this even if constructed this way.